gemrb/core/Scriptable/Actor.cpp

```diff
@@ -6305,7 +6305,7 @@ bool Actor::ValidTarget(int ga_flags, const Scriptable *checker) const
 	}
 	if (ga_flags & GA_ONLY_BUMPABLE) {
 		if (core->InCutSceneMode()) return false;
-		if (core->GetGame()->CombatCounter) return false;
+		if (core->GetGame()->CombatCounter && !InParty) return false;
```

Comment:
so you can also bump summons:
```suggestion
		if (core->GetGame()->CombatCounter && !IsPartyMember()) return false;
```

Reply:
wait, that's still odd. Why would you allow bumping for anyone in combat?

Reply:
It wasn't strictly related to bumping, but to the in-combat deadlocks when an actor was bumping itself. I found a smarter way though, which only avoids self-bumping.